promptsource/templates/anli/templates.yaml

```diff
@@ -1,27 +1,38 @@
 dataset: anli
 templates:
-  391acb43-732a-4c09-8c10-e02d226d8854: !Template
+  0cc3ae39-3997-4686-8c93-5d51457efa1f: !Template
     answer_choices:
-    - 'No'
-    - Neutral
-    - 'Yes'
+    - correct
+    - inconclusive
+    - incorrect
     answer_choices_key: null
-    id: 391acb43-732a-4c09-8c10-e02d226d8854
-    jinja: 'Sentence 1: {{premise}}
-
-      Sentence 2: {{hypothesis}}
-
-      Question: Does Sentence 1 contradict Sentence 2? Yes, No, or Neutral? |||
-
-      {{answer_choices[label]}}'
+    id: 0cc3ae39-3997-4686-8c93-5d51457efa1f
+    jinja: '{{premise}} Using only the above description and what you know about the
+      world, "{{hypothesis}}" is definitely correct, incorrect, or inconclusive? |||
+      {{ answer_choices[label] }}'
     metadata: !TemplateMetadata
-      _do_eval: true
-      _do_train: false
-      choices_in_prompt: null
-      metrics: []
-      original_task: false
-    name: does S1 contradict S2?
-    reference: Copied from Victor's prompts for XNLI.
+      choices_in_prompt: true
+      metrics:
+      - Accuracy
+      original_task: true
+    name: MNLI crowdsource
+    reference: Adapted from Williams et al. 2018's instructions to crowdsourcing workers.
+  179eb863-3ece-4e6f-af0f-fcb46d997306: !Template
+    answer_choices:
+    - 'yes'
+    - maybe
```

Comment:
any idea why sometimes the answer_choices have quotes and sometimes they don't?

Reply:
I had the same question. Checked YAML's spec and it says quotes don't matter unless there are colons in the string. So probably doesn't matter for now; should be enforced from streamlit's side.